os/arch/arm/src/armv7-a/arm_cpustart.c

```diff
@@ -111,6 +112,9 @@ int arm_start_handler(int irq, void *context, void *arg)
 
 	svdbg("CPU%d Started\n", this_cpu());
 
+	/* start generic timer PPI for this core */
+	up_timer_initialize();
+
```

Comment:
I think we must call up_timer_disable(); here.

Reply:
When I tested your latest code, the flash write is still causing issue. 
When the write is happening on cpu0, then time is ok (11 secs).
But when running on cpu1, the time shows 28secs.

```
TASH>>hello 0
TASH>>Hello, World!!
Running file write test on cpu 0
BEFORE: (1747612937 sec, 136000000 nsec)
AFTER:  (1747612948 sec, 763000000 nsec)
Time taken = 11 secs


TASH>>hello 1
TASH>>Hello, World!!
Running file write test on cpu 1
BEFORE: (1747612934 sec, 801000000 nsec)
AFTER:  (1747612962 sec, 222000000 nsec)
Time taken = 28 secs
```